src/partials/head-scripts.hbs

```diff
@@ -1,5 +1,24 @@
-    {{#with site.keys.googleAnalytics}}
-    <script async src="https://www.googletagmanager.com/gtag/js?id={{this}}"></script>
-    <script>function gtag(){dataLayer.push(arguments)};window.dataLayer=window.dataLayer||[];gtag('js',new Date());gtag('config','{{this}}')</script>
-    {{/with}}
-    {{!-- <script>var uiRootPath = '{{{uiRootPath}}}'</script> --}}
+{{#with site.keys.googleAnalytics}}
+<script async src="https://www.googletagmanager.com/gtag/js?id={{this}}"></script>
+<script>function gtag() { dataLayer.push(arguments) }; window.dataLayer = window.dataLayer || []; gtag('js', new Date()); gtag('config', '{{this}}')</script>
+{{/with}}
+{{!--
+<script>var uiRootPath = '{{{uiRootPath}}}'</script> --}}
+
```

Comment:
Preloading the fonts avoids the initial flicker when loading a new page. The icons were especially bad, and this fixes it.